### DISCUSSION TITLE
100.000 IBXC How will the 7 lucky winners be chosen?

### DISCUSSION THREAD

**Discussion by ergun818** [IBAX-io/go-ibax#1285] [Rewards]
- Randomly pick 7 lucky participants who exceed 45,000 points will receive 100,000 IBXC.
How will this choice be made among users who exceed 45,000 points?
How can we reach 45,000 points?
I have 26,000 points yet :(

**Comment by abuzercan** [comment]
nasıl puan kasarız hiç fikrim yok 30k da kaldım

**Reply by elonunmk** [reply]
I hope my efforts will not be wasted. I have great expectations
